validation/requirements.yaml

```diff
@@ -0,0 +1,358 @@
+CFG-R001:
+  description: BBI config JSON gets created
+  tests:
+  - INT-CFG-001
+CFG-R002:
+  description: Config values are correct in written file
+  tests:
+  - INT-CFG-002
+COVCOR-R001:
+  description: CovCor parses .cov and .cor files
+  tests:
+  - INT-COVCOR-001
+COVCOR-R002:
+  description: CovCor errors with bad input
+  tests:
+  - INT-COVCOR-002
+DATA-R001:
+  description: Has valid data path for CTL
+  tests:
+  - INT-DATA-001
+DATA-R002:
+  description: Fails with invalid data path
+  tests:
+  - INT-DATA-002
+DATA-R003:
+  description: Has valid complex path CTL And Mod
+  tests:
+  - INT-DATA-003
+EXP-R001:
+  description: BBI expands without prefix
+  tests:
+  - INT-EXP-001
+EXP-R002:
+  description: BBI expands with prefix
+  tests:
+  - INT-EXP-002
+EXP-R003:
+  description: BBI expands with prefix to partial match
+  tests:
+  - INT-EXP-003
+INIT-R001:
+  description: BBI can initialize
+  tests:
+  - INT-INIT-001
+LOCAL-R001:
+  description: BBI completes local execution
+  tests:
+  - INT-LOCAL-001
+LOCAL-R002:
+  description: NMFE options end in script
+  tests:
+  - INT-LOCAL-002
+LOCAL-R003:
+  description: BBI can execute in parallel
+  tests:
+  - INT-LOCAL-003
+LOCAL-R004:
+  description: BBI loads default config
+  tests:
+  - INT-LOCAL-004
+LOCAL-R005:
+  description: BBI loads config specified by absolute path
+  tests:
+  - INT-LOCAL-005
+LOCAL-R006:
+  description: BBI loads config specified by relative path
+  tests:
+  - INT-LOCAL-006
+NMQ-R001:
+  description: NMQUAL execution succeeds
+  tests:
+  - INT-NMQ-001
+NMQ-R002:
+  description: Hashing for NMQual works with original Mod file
+  tests:
+  - INT-NMQ-002
+PARAM-R001:
+  description: Parse params for single model
+  tests:
+  - INT-PARAM-001
+PARAM-R002:
+  description: Parse params for directory of models
+  tests:
+  - INT-PARAM-002
+POSTEX-R001:
+  description: KVP expansion works
+  tests:
+  - INT-POSTEX-001
+POSTEX-R002:
+  description: Post execution succeeds
+  tests:
+  - INT-POSTEX-002
+SGE-R001:
+  description: BbiCompletesSGEExecution
+  tests:
+  - INT-SGE-001
+SGE-R002:
+  description: BBBI completes parallel SGE execution
```

Comment:
s/BBBI/BBI/

Reply:
When you run it in parallel it gets an extra B.